frontend/coreproject/src/lib/components/pages/anime_info/AnimeMainDetails.svelte

```diff
@@ -0,0 +1,156 @@
+			<div>
+				<h2 class="font-bold 2xl:!text-6xl">{anime.titles.eng}</h2>
+
+				<div class="anime_titles mt-3 flex items-center gap-2 md:text-xs 2xl:!text-lg font-medium opacity-80">
+					<span>{anime.titles.japanese}</span>
+					<span>-</span>
+					<span>{anime.titles.eng}</span>
+					<span>-</span>
+					<span>{anime.titles.others}</span>
+				</div>
+
+				<div
+					class="anime_basic_details mt-2 flex flex-wrap items-center gap-2 pr-5 md:text-xs 2xl:!text-lg 2xl:!leading-none font-semibold opacity-100"
+				>
+					<span>{anime.type}</span>
+					<span>-</span>
+					<span>{anime.episodes} eps</span>
+					<span>-</span>
+					<span>{anime.status}</span>
+					<span>-</span>
+					<span>{anime.premiered}</span>
+					<span>-</span>
+					<span>{anime.studio}</span>
+				</div>
+
+				<div class="md:mt-5 2xl:mt-10 flex items-center gap-5">
+					<button
+						type="button"
+						class="btn 2xl:btn-lg md:h-16 2xl:h-24 justify-start md:rounded-lg 2xl:rounded-xl bg-primary-500 md:text-xs 2xl:text-xl font-bold text-white"
+					>
+						<PlayCircle width="30" height="30" color="white" />
+						<div class="flex flex-col items-start">
+							<span class="leading-5">Watch</span>
+							<span class="md:text-[0.6rem] 2xl:text-lg font-normal leading-none opacity-70">Ep 01</span>
+						</div>
+					</button>
+
+					<button
+						type="button"
+						class="btn md:h-16 2xl:h-24 2xl:w-24 flex-col gap-1 md:rounded-lg 2xl:rounded-xl bg-secondary-100 md:text-xs 2xl:text-xl md:font-bold 2xl:font-semibold text-surface-500"
+					>
+						<Read width="22" height="22" color="bg-surface-500" />
+						Read
+					</button>
+
+					<button
+						type="button"
+						class="btn md:h-16 2xl:h-24 2xl:w-24 flex-col gap-1 md:rounded-lg 2xl:rounded-xl bg-secondary-100 md:text-xs 2xl:text-xl md:font-bold 2xl:font-semibold text-surface-500"
+					>
+						<Listen width="22" height="22" color="bg-surface-500" />
+						Listen
+					</button>
+				</div>
+
+				<div class="user_options mt-5 flex md:gap-2 2xl:gap-3">
+					<button
+						type="button"
+						class="btn-icon md:btn-icon-sm 2xl:btn-icon-lg md:rounded 2xl:rounded-lg bg-warning-400 p-0 text-surface-500"
+					>
+						<Video width="20" />
+					</button>
+
+					<button
+						type="button"
+						class="btn-icon md:btn-icon-sm 2xl:btn-icon-lg md:rounded 2xl:rounded-lg bg-warning-400 p-0 text-surface-500"
+					>
+						<Edit width="20" variant="with_underline_around_pencil" />
+					</button>
+
+					<button
+						type="button"
+						class="btn-icon md:btn-icon-sm 2xl:btn-icon-lg md:rounded 2xl:rounded-lg bg-warning-400 p-0 text-surface-500"
+					>
+						<Download width="20" />
+					</button>
+
+					<button
+						type="button"
+						class="btn-icon md:btn-icon-sm 2xl:btn-icon-lg md:rounded 2xl:rounded-lg bg-warning-400 p-0 text-surface-500"
+					>
+						<Share width="20" />
+					</button>
```

Comment:
These should be close to `36px`

![image](https://user-images.githubusercontent.com/61817579/230699913-b1c8c0a0-17bc-4ab1-b172-cb3c4d93a00d.png)

Reply:
What you mean by this
> These should be close to 36px
Button size ?

Reply:
> What you mean by this

Btn is larger than desgin. Button should be 36px in height and width